### PR TITLE
register class creation #54

### LINKED ISSUE #54: Registro de la clase forecasts
Hacer una clase para guardar datos de la clase forecasts #9

### PR DESCRIPTION
**Pull request by sergiohut**
I´ve created a class Register in order to store all the Climbing-crag-forecast that were made so they can be checked after closes #54 related with #9

### CODE REVIEW

**Review by DaniWalkington — APPROVED** [review]
LGTM 👍

**Comment by sergiohut** [comment]
@jj revisar

**Comment by JJ** [comment]
> I´ve created a class Register in order to store all the Climbing-crag-forecast that were made so they can be checked after

Solving which issue?

**Comment by JJ** [comment]
@VickyTinar when you assign PRs to someone, it's to *review*, not to carry them out. It's issues what you should assign to someone in order to make them.

**Comment by sergiohut** [comment]
@jj solving issue #54, it is now included in the PR description

**Review by VickyTinar — APPROVED** [review]

File: lib/register.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+       * @type {ClimbingCragForecast[]}
+       */
+      this.climbingCragforecasts = [];
+    }
+  
+    /**
+     * Adds a climbing crag forecast to the register.
+     * @param {ClimbingCragForecast} forecast - The climbing crag forecast to add.
+     */
+    addForecast(forecast) {
```
Comment (JJ) [inline lib/register.js:17]: Are you sure milestone 0 requires functions?

**Review by JJ — COMMENTED** [review]
Is `Register` a name that is shared by the customer?

**Review by dinho-dev — APPROVED** [review]
LGTM

**Comment by sergiohut** [comment]
we finally changed the class name so it is more descriptive, also removed the method of this class since it is not featured in the milestone 0 @jj revision

**Pull request merged**